.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         run: npm ci
 
       - name: Build docs
-        run: mkdocs build --strict
+        run: npm run build
```

Comment:
Personally, I'd like to keep this the same as Netlify. So, let's wait after #1194 is done.